Source/MonoGame.Extended/Collections/Poolable.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace MonoGame.Extended.Collections
+{
+    public abstract class Poolable : IPoolable
```

Comment:
Cool. It's decided then. As I said I'll merge it in about 3 hours when I get home. If you want to make any more changes before then (e.g. delete commented code) otherwise all good.